### PR TITLE
Performance tests (read/write/list) for HNS bucket

### PR DESCRIPTION
**Pull request by anushka567**
### Description
Modified existing script to run performance tests on HNS as well as flat bucket and upload metrics to [gsheet](https://docs.google.com/spreadsheets/d/1wXRGYyAWvasU8U4KaP7NGPHEvgiOSgMd1sCLxsQUwf0/edit?usp=sharing&resourcekey=0-Kv_559oGD9Z9474dPN6dxA).
### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Ran the scripts manually for a smaller workload.
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh [hunk 1]
```diff
@@ -39,17 +39,38 @@ then
...
+touch config.yml
+echo "enable-hns: true
+metadata-cache:
+  ttl-secs: 0" > config.yml
+COMMON_MOUNT_FLAGS="--debug_fuse --debug_gcs --log-format \"text\""
+GCSFUSE_LS_FLAGS="--config-file=../config.yml $COMMON_MOUNT_FLAGS"
+LOG_FILE_FIO_TESTS=${KOKORO_ARTIFACTS_DIR}/gcsfuse-logs-hns.txt
+GCSFUSE_FIO_FLAGS="--config-file=config.yml $COMMON_MOUNT_FLAGS --log-file $LOG_FILE_FIO_TESTS --stackdriver-export-interval=30s"
+BUCKET_NAME="periodic-perf-tests-hns"
+SPREADSHEET_ID='1wXRGYyAWvasU8U4KaP7NGPHEvgiOSgMd1sCLxsQUwf0'
```
Comment (ankitaluthra1) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:72]: what is this hard coded id used for ?

Comment (anushka567) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:72]: We have existing spreadsheet where the fio and ls metrics are getting uploaded  daily from the periodic tests, 
So for the flat bucket the metrics are getting uploaded(first hard coded id) [here](https://docs.google.com/spreadsheets/d/1kvHv1OBCzr9GnFxRu9RTJC7jjQjc9M4rAiDnhyak2Sg/edit?gid=0#gid=0).
For the hns bucket metrics i have created another copy of similar gsheet where the results will be uploaded. [Here](https://docs.google.com/spreadsheets/d/1wXRGYyAWvasU8U4KaP7NGPHEvgiOSgMd1sCLxsQUwf0/edit?resourcekey=0-Kv_559oGD9Z9474dPN6dxA&gid=0#gid=0). (currently this contains the results from the smaller workload that i used for testing the script)

Comment (ankitaluthra1) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:72]: okay,so new runs will override the old result and add latest results in same spreadsheet every time...this spreadsheet link is provided in kokoro pipeline?

Comment (anushka567) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:72]: Yes.

**Review by ankitaluthra1 — APPROVED** [review]
For long term we need to have a way to create new gsheet on every run, not blocking this PR on this.

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh [hunk 2]
```diff
@@ -39,17 +39,38 @@ then
   UPLOAD_FLAGS="--upload_gs"
 fi
 
+run_load_test_and_fetch_metrics(){
+  # Executing perf tests
+  ./run_load_test_and_fetch_metrics.sh "$1" "$2" "$3" "$4" "$5"
```
Comment (Tulsishah) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:44]: Add comment what "$1" "$2" "$3" "$4" "$5" refer

Comment (Tulsishah) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:44]: Better to add const 
e.g.
X=$1

**Review by Tulsishah — APPROVED** [review]
Please confirm if existing script is working fine.

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh (hunk 2 above)
Comment (anushka567) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh]: Added.

**Review by Tulsishah — APPROVED** [review]

**Review by Tulsishah — APPROVED** [review]

**Pull request merged**